Random failure due to "unreliable" results in emulation test

I had a random intermittent failure on CircleCI for `external/fv3fit/tests/emulation/thermobasis/test_emulator.py[test_Config_multi_output_levels]`:

```
E   hypothesis.errors.Flaky: Hypothesis test_Config_multi_output_levels(levels=[32007, 7175298582426376939, 117, 29170, 4235]) produces unreliable results: Falsified on the first call but did not on a subsequent one
```

When I reran without changing anything, the test passed. Has anyone run into this before or is it a super rare incident? @nbren12 @frodre 

https://app.circleci.com/pipelines/github/ai2cm/fv3net/10033/workflows/d11d7bcd-88b5-4e2f-a076-fe0bf8e1ad53/jobs/87314

--- a/external/fv3fit/tests/emulation/thermobasis/test_emulator.py
+++ b/external/fv3fit/tests/emulation/thermobasis/test_emulator.py
@@ -151,16 +151,13 @@ def test_Config_register_parser(args, loss_cls):
         assert not config.relative_humidity
 
 
-@given(lists(integers(min_value=0)))
+@given(lists(integers(min_value=0, max_value=100)))
 def test_Config_multi_output_levels(levels):
-
     str_levels = ",".join(str(s) for s in levels)
-
     parser = argparse.ArgumentParser()
     Config.register_parser(parser)
     args = parser.parse_args(["--multi-output", "--levels", str_levels])
     config = Config.from_args(args)
-
     assert config.target.levels == levels
 
 

